tests/features/wiki_communities.feature

```diff
@@ -0,0 +1,51 @@
+@api @javascript @communities
+Feature: Wiki
+  In order to allow users to create and use the wiki on communities profile
+  As a site administrator
+  I want to make sure auhtenticated users with proper permissions can create wiki pages on their communities
+
+  Background:
+    Given users:
+      | name          | mail                 | pass         | roles         |
+      | administrator | admin@ecample.com    | password123  | administrator |
+      | contribuser   | contrib@example.com  | password456  | contributor   |
+    And these featureSet are enabled
+      | featureSet     |
+      | Wiki           |
+    And 'Community' content:
+      | title             | author        | workbench_moderation_state_new | workbench_moderation_state | language | status |
+      | Community Example | administrator | published                      | published                  | en       | 1      |
+
+  Scenario: An administrator user can create a wiki on a community
+    When I am logged in as "administrator"
+    And I use device with "1920" px and "1080" px resolution
+    And I go to "community/community-example"
+    And I click "Create content"

```

Comment:
This line is failing on ec_europa. I suggest you check the HTML generated with the ec_europa theme to make sure you can access it the same way.